### PR TITLE
WIP: actionWrap

### PR DESCRIPTION
**Pull request by programmersstudio**

### CODE REVIEW

File: deployer/src/deploy-struct.ts [hunk 1]
```diff
@@ -131,6 +131,7 @@ export interface DeployStructure {
     cleanNamespace?: boolean // Clears entire namespace prior to deploying
     bucket?: BucketSpec // Information guiding deployment of web resources into an s3 (or s3-like) object store bucket
     actionWrapPackage?: string // The name of a package into which web resources will be action-wrapped.
+    actionWrapPackageWithRouter?: boolean // The name of a package into which web resources will be action-wrapped.
```
Comment (rabbah) [inline deployer/src/deploy-struct.ts:134]: I don’t think we need a control flag that’s user visible. We should just do it this way by default.

Comment (programmersstudio) [reply deployer/src/deploy-struct.ts:134]: ok, will make the changes.

Comment (rabbah) [reply deployer/src/deploy-struct.ts:134]: There is a question of routing we might need to settle. Will follow up briefly in slack first.

**Comment by joshuaauerbachwatson** [comment]
As discussed.   Shelving this direction for now.

**Pull request closed without merging**